apps/block_scout_web/assets/css/theme/_variables.scss

```diff
@@ -1,24 +1,5 @@
-@import "theme/base_variables";
-@import "neutral_variables";
-// @import "dai_variables";
-// @import "ethereum_classic_variables";
-// @import "ethereum_variables";
-// @import "ether1_variables";
-// @import "expanse_variables";
-// @import "gochain_variables";
-// @import "goerli_variables";
-// @import "kovan_variables";
-// @import "lukso_variables";
-// @import "musicoin_variables";
-// @import "pirl_variables";
-// @import "poa_variables";
-// @import "posdao_variables";
-// @import "rinkeby_variables";
-// @import "ropsten_variables";
-// @import "social_variables";
-// @import "sokol_variables";
-// @import "tobalaba_variables";
-// @import "tomochain_variables";
```

Comment:
it would be better to not remove the comments for the next one who's going to merge with upstream. Please try to keep changes to a minimum next time.